src/test/resources/database/book/add-three-books.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO books (id, title, author, isbn, price, description, coverImage)
+VALUES (1, 'Harry Potter', 'Joanne Rowling', '12345', 50.00, 'Book about boy...', 'img.harry.potter');
+
+INSERT INTO books (id, title, author, isbn, price, description, coverImage)
+VALUES (2, 'The little prince', 'Antoine de Saint-Exupery', '4557839', 80.00, 'Story about little prince', 'img.little.prince');
+
+INSERT INTO books (id, title, author, isbn, price, description, coverImage)
+VALUES (3, 'The hobbit', 'J. R. R. Tolkien', '93587', 40.00, 'Book about hobbit Bilbo', 'img.hobbit');
```

Comment:
add an empty line everywhere you see this
![image](https://github.com/user-attachments/assets/e5d13a68-43d0-454d-b332-895fb5b68c18)